Cargo.toml

```diff
@@ -21,7 +21,7 @@ license = "GPL-3.0-or-later"
 categories = ["cryptography"]
 
 [lib]
-crate-type = ["lib"]
+crate-type = ["staticlib", "rlib"]
```

Comment:
why this change is needed? what is the implication?

Reply:
iOS must be build with staticlib as too many dependencies would be missing, and x86_64 host should be fine anyway (faster static libs with slower loading times due to a bigger size vs smaller but slower dynamic libs with faster loading times)  but... I just realized all platform-specific overrides can be placed in dinghy.toml!! - will fix that asap

Reply:
moved this from Cargo.toml to dinghy.toml, don't see why GitHub not showing latest changes :|